lib/jxl/enc_modular.cc

```diff
@@ -1417,6 +1417,7 @@ Status ModularFrameEncoder::PrepareStreamParams(const Rect& rect,
       case SpeedTier::kKitten:
         nb_rcts_to_try = 9;
         break;
+      case SpeedTier::kGlacier:
```

Comment:
Could consider to try even more RCTs. IIRC, the 19 are the unique ones up to permutation, but with the extra properties, permutations could make a difference. Probably not worth it but it might be interesting to check how much of a difference it can make...